x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/translations.ts

```diff
@@ -35,6 +35,21 @@ export const ALERT_DETAILS = i18n.translate(
   }
 );
 
+export const TECHNICAL_PREVIEW = i18n.translate(
+  'xpack.securitySolution.timeline.expandableEvent.technicalPreviewLabel',
+  {
+    defaultMessage: 'Technical Preview',
+  }
+);
+
+export const TECHNICAL_PREVIEW_DESCRIPTION = i18n.translate(
+  'xpack.securitySolution.timeline.expandableEvent.technicalPreviewDescription',
+  {
+    defaultMessage:
+      'This functionality is in technical preview and may be changed or removed completely in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.',
```

Comment:
I'm assuming this is standard verbiage for tech previews, but if not, I would suggest running the text past @caitlinbetz or maybe our Docs team.

Reply:
I believe it is a standard text for Tech Preview badges